_profiles/works.html

```diff
@@ -25,7 +25,7 @@
   connectivity: "Cable"
   location: "London_EC2"
   runs: 1
-  url: "https://next.wellcomecollection.org/works?query=david&cohort=platformTeam"
```

Comment:
Do we need the cohort now?

Reply:
Just using it to test worst case scenario (which is probably more common), where images aren't served by the CDN

Reply:
Ah, okay, I didn't realise this cohort had this as a feature.